start.sh

```
@@ -22,9 +22,9 @@ fi
   -e "s:<NGINX_CLIENT_MAX_BODY_SIZE>:${NGINX_CLIENT_MAX_BODY_SIZE:-5M}:g" \
   /etc/nginx/nginx.conf.template > /etc/nginx/nginx.conf
 
-# Wait for the application to start before accepting ALB requests.
 while sleep 2; do
-  curl --verbose --fail --max-time 5 "http://${APP_HOST:-app}:${APP_PORT}${HEALTHCHECK_PATH:-/health}" && break
+  wget --spider "http://${APP_HOST:-app}:${APP_PORT}${HEALTHCHECK_PATH:-/health}" || continue
```

The version of `wget` in busybox is very stripped down so it doesn't have all the bells and whistles of curl. That said we can emulate the behaviour we are after by telling `wget` to just check the "file" (our endpoint) exists.

OR the command because it will return a non-zero exit code if it cannot find the resource and cause the entire script, to fail, but the conditional here swallows that error code allowing us to continue.